cmd/initializer/Dockerfile

```diff
@@ -11,8 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+FROM golang:1.19.2 as builder
```

Comment:
`1.20`?

And is there any concern about the docker.io rate limit affecting CI jobs? I see a few other places within the kubernetes org that depend on this repo, but I don't have context on any of those.

Reply:
Thanks Ben. 
As for docker.io rate limits, this particular repo doesn't get built too often. I tried searching for an alternative on registry.k8s.io, but nothing stood out.